release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1077,6 +1077,13 @@ With this release, control plane machine sets are supported for clusters that ru
 
 For more information, see xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-getting-started[Getting started with the Control Plane Machine Set Operator].
 
+[NOTE]
+====
+For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version} and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the related link:https://access.redhat.com/solutions/7013893[knowledge base article].
```

Comment:
It might be unavoidable, but the first sentence is very long and hard to follow. Is it possible to break it up a bit more? Or perhaps is the second instance of "and have root volume availability zones" not required?

Reply:
Trying to keep things the same as the known issues later on, but yeah, this is not ideal.

```suggestion
For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version}, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in link:https://access.redhat.com/solutions/7013893[OpenShift on OpenStack with Availability Zones: Invalid Compute ServerGroup setup during OpenShift deployment].
```